packages/container/tests/container_test.js

```diff
@@ -15,6 +15,19 @@ QUnit.module('Container', {
   }
 });
 
+function lookupFactory(name, container, options) {
+  let factory;
+  if (isFeatureEnabled('container-factoryFor')) {
+    expectDeprecation(() => {
```

Comment:
There is also an `ignoreDeprecation` which might be appropriate/useful here. Example:

https://github.com/emberjs/ember.js/blob/aeeddf7c00b42a340494465252481b5414d87009/packages/ember-runtime/tests/controllers/controller_test.js#L118